app/javascript/components/mentoring/mentor_inbox/sorting.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+const sortingOptions = [
+  { value: 'exerciseTitle', description: 'Sort by Exercise' },
+  { value: 'menteeHandle', description: 'Sort by Student' },
+]
```

Comment:
I've hardcoded the sorting options here. We could pass them as parameters, although I probably wouldn't want to pass their description too as that is a UI responsibility.